### PR TITLE
patp now uses ~, and inter on nickname

### PR DESCRIPTION
**Pull request by jyng**
Screen Shot 2020-02-10 at 7 38 51 PM
Screen Shot 2020-02-10 at 7 38 25 PM

Would be curious if there's a better way to write this logic @loganallenc @matildepark

### CODE REVIEW

File: pkg/interface/chat/src/js/components/lib/message.js [hunk 1]
```diff
@@ -167,10 +167,10 @@ export class Message extends Component {
       let color = "#000000";
       if (contact) {
         name = (contact.nickname.length > 0)
-          ? contact.nickname : props.msg.author;
+          ? contact.nickname : "~" + props.msg.author;
         color = `#${uxToHex(contact.color)}`;
-      } 
-
+        console.log(name);
```
Comment (matildepark) [inline pkg/interface/chat/src/js/components/lib/message.js]: This can go.

Comment (jyng) [reply pkg/interface/chat/src/js/components/lib/message.js]: lmao my bad

File: pkg/interface/chat/src/js/components/lib/message.js [hunk 2]
```diff
@@ -167,10 +167,10 @@ export class Message extends Component {
       let color = "#000000";
       if (contact) {
         name = (contact.nickname.length > 0)
```
Comment (matildepark) [inline pkg/interface/chat/src/js/components/lib/message.js]: Cleaner? Idk, you can do \`~${props.msg.author}\` if you use backtick quotes like that, I guess.

**Comment by matildepark** [comment]
But honestly this looks concise enough; I wouldn’t necessarily worry about making it more and more clean. Inline is best unless it detracts from legibility or has excessive logic, in which case you’d start refactoring it outside the flow or in its component (if it’s a component, anyway). This is fine inline and the ternaries are basic.

File: pkg/interface/chat/src/js/components/lib/message.js (hunk 2 above)
Comment (jyng) [reply pkg/interface/chat/src/js/components/lib/message.js]: Hmm, could wee a variable that prepends the sig? This would make it easier to QA.

Comment (matildepark) [reply pkg/interface/chat/src/js/components/lib/message.js]: (Idk why my phone made it do this a line too high.) Honestly this is fine as is imo but I’ll wait and see what the other reviewers think too.

**Comment by jyng** [comment]
Ok will send this PR to the boss person

**Comment by loganallenc** [comment]
Jimmy, please run gulp bundle-prod to minify these files, and rebase to remove the "large unminified versions". Otherwise looks good.

File: pkg/interface/chat/src/js/components/lib/message.js (hunk 1 above)
Comment (loganallenc) [inline pkg/interface/chat/src/js/components/lib/message.js]: remove this console.log

File: pkg/interface/chat/src/js/components/lib/message.js (hunk 2 above)
Comment (loganallenc) [reply pkg/interface/chat/src/js/components/lib/message.js]: I prefer Matilde's version of this as well

File: pkg/interface/chat/src/js/components/lib/message.js (hunk 1 above)
Comment (matildepark) [reply pkg/interface/chat/src/js/components/lib/message.js]: I think he removed it already in 8f580aa3bb4e275e433bfe2dcd0c17d8e5f95faa

**Comment by jyng** [comment]
Do I have to rebase to remove? You just want to me to minify the files in `dist/` right?

**Comment by loganallenc** [comment]
Merged and done. Thanks!

**Pull request closed without merging**